ansible/roles/pgo-operator/files/pgo-configs/pgbouncer-template.json

```diff
@@ -44,10 +44,20 @@
 		    {{.ContainerResources }}
                     "env": [{
                         "name": "PG_USERNAME",
-                        "value": "{{.PgBouncerUser}}"
+                        "valueFrom": {
+                            "secretKeyRef": {
+                                "name": "{{.SecretsName}}",
```

Comment:
Can we take the time to make these names more understandable in context? i.e. 'SecretsName" could be PGSecretsName or XYXSecretsName. Understand it has been this way, but I think the change will help. Comment applies to anywhere "SecretsName" or equally generic naming in the template(s) is/are used.

Reply:
Done - let me know if those names don't work